test-integration/helpers/window-position.coffee

```diff
@@ -0,0 +1,12 @@
+windowPosition = (test) ->
```

Comment:
Not really ;)  This and `wait` both have the same usage, i.e. `fn(test).action()`  They're just implemented differently.  My (fairly uninformed) thoughts were that the class syntax is better for larger helpers since it keeps the state enclosed, while the closure is better for simple ones like this one.  It doesn't really matter much to me though.